changelogs/api-management-service.md

```diff
@@ -2,6 +2,28 @@
 
 Developer portal follows an independent release lifecycle and the [per-release changelog is available in the developer portal's GitHub repository](https://aka.ms/apimdevportal/releases).
 
+## Release - API Management service: January, 2022
+
+A regular Azure API Management service update was started on January 20, 2021, and included the following new features, bug fixes, and other improvements. It may take several weeks for your API Management service to receive the update.
```

Comment:
We are in  `2022` :)

Reply:
Every year, the same problem: https://github.com/Azure/API-Management/commit/7e913caf468ca4eba8f49af1b28edb836f0c7652